pyvista/core/filters/uniform_grid.py

```diff
@@ -116,3 +116,57 @@ def extract_subset(self, voi, rate=(1, 1, 1), boundary=False, progress_bar=False
         fixed.field_data.update(result.field_data)
         fixed.copy_meta_from(result)
         return fixed
+
+    def image_fft(self, progress_bar=False):
```

Comment:
It'd be nice to have an option to override the name instead of the default `ImageScalars`

Reply:
Implemented.